files/ko/web/javascript/reference/global_objects/null/index.html

```diff
@@ -42,28 +52,20 @@ <h3 id="null과_undefined의_차이"><code>null</code>과 <code>undefined</code>
 isNaN(1 + null)      // false
 isNaN(1 + undefined) // true</pre>
 
-<h2 id="명세">명세</h2>
-
-<table class="standard-table">
- <tbody>
-  <tr>
-   <th scope="col">명세</th>
-  </tr>
-  <tr>
-   <td>{{SpecName('ESDraft', '#sec-null-value', 'null value')}}</td>
-  </tr>
- </tbody>
-</table>
-
-<h2 id="브라우저_호환성">브라우저 호환성</h2>
+<h2 id="Specifications">명세</h2>
 
+{{Specifications}}
 
+<h2 id="Browser_compatibility">브라우저 호환성</h2>
 
-<p>{{Compat("javascript.builtins.null")}}</p>
+<p>{{Compat}}</p>
 
-<h2 id="같이_보기">같이 보기</h2>
+<h2 id="See_also">같이 보기</h2>
 
 <ul>
- <li>{{jsxref("undefined")}}</li>
- <li>{{jsxref("NaN")}}</li>
-</ul>
+  <li>{{jsxref("undefined")}}</li>
+  <li>{{jsxref("NaN")}}</li>
+  <li>
+    <p>{{jsxref("Operators/void", "void operator")}}</p>
```

Comment:
"operator" 번역이 누락됐어요.

```suggestion
    {{jsxref("Operators/void", "void")}} 연산자
```

불필요한 `<p>`랑, "operator"까지 `<code>` 안에 들어가는건 영어 소스가 잘못됐네요.